README.md

```diff
@@ -226,9 +231,11 @@ The CLI is interrupted after tests began executing on instances, then resumed by
 ### Table Headers
 
 * `INSTANCE TYPE`: instance type
-* `MEETS TARGET UTILIZATION?`: SUCCESS if max CPU and max MEM are less than target utilization; FAIL otherwise
+* `STATUS`: SUCCESS if max CPU and max MEM are less than their respective thresholds; FAIL otherwise
 * `CPU_USAGE_ACTIVE`: max `cpu_usage_active` recorded (p100) over the duration of instance-qualifier run
+* `THRESHOLD`: cpu threshold set by user
```

Comment:
maybe make this one 'CPU_THRESHOLD' and the other one 'MEM_THRESHOLD' ?

Reply:
Yeah, I was trying to make the table narrower but definitely not worth at the cost of clarity. will address in revision